.github/workflows/release_aarch64.yml

```diff
@@ -0,0 +1,50 @@
+name: PyPi Linux aarch64 release
```

Comment:
Why a different release file? will the upload work for new releases with two workflows?

Reply:
Yes two different workflow, [release.yml](https://github.com/acsylla/acsylla/blob/update_build_environment/.github/workflows/release.yml)  i just update for use fresh build environment [build_manylinux2014_wheel.sh](https://github.com/acsylla/acsylla/blob/update_build_environment/bin/build_manylinux2014_wheel.sh) and  [release_aarch64.yml](https://github.com/acsylla/acsylla/blob/update_build_environment/.github/workflows/release_aarch64.yml) added for build pypi packages for aarch64 (this pipeline work very slow, because build in docker via qemu)

Reply:
Please try to build the release, if everything works as expected, then we can merge them into one pipeline. But I hope github will add soon native build environment for aarch64 and we rewrite this pipeline

Reply:
lets merge it and lets test it